src/screens/LoginScreen.tsx

```diff
@@ -59,19 +58,19 @@ const LoginScreen = ({
         addressesToInitialize = await deriveWalletStoredAddresses(wallet)
 
         dispatch(walletSwitched({ wallet, addressesToInitialize, pin }))
-        restoreNavigationState(true)
+        setNavigationState()
```

Comment:
Nit: what about setting it to `null` explicitly? Or better yet (IMO), make `setNavigationState` accept either a new state as argument (as it is now), OR the string 'initial' to make it clear that we want to reset the state. Or better yet (IMO) 🙃 to use a `resetNavigationState` function?

Reply:
hehe alright, I'll make it more explicit! I agree with you